src/map.js

```diff
@@ -47,9 +47,9 @@ export default function MapComponent({ searchedLocation }) {
       const marker = L.marker(searchedLocation).addTo(map);
 
       // add a popup to the marker
-      marker.bindPopup("Searched Location").openPopup();
+      marker.bindPopup(searchedLocationName).openPopup();
     }
-  }, [map, searchedLocation]);
+  }, [map, searchedLocation, searchedLocationName]);
```

Comment:
I think here we shouldn't add the map as a dependency because it will be re rendered every second,
in my PR I am just checking for the searchedLocation if it changes I re render the map

Reply:
Originally I wanted to use searchedLocation as my prop so that bindPopup would show the name of the city after the search was conducted however, every attempt resulted in a appendChild error. Is there a better way that you know of that I can implement to clean this up?

Reply:
Please add the final outcome of the discussion and please resolve the merge conflicts @Doej0

Reply:
David and I have agreed that no changes will need to be made, my addition will not hinder rendering as my PR request does not reflect the updated changes David made.